triton-server/src/notification.ts

```diff
@@ -141,15 +139,13 @@ const mockDataset: TritonDataset = {
     latest_release_update: new Date(),
 }
 
-const formatDateAndTime = (date: Date): string => {
-    const cleanedDate = new Date(date)
+const formatDateAndTime = (): string => {
+    const cleanedDate = new Date()
+    const hours = new Date().getHours() - 1
+    cleanedDate.setHours(hours)
     return (
         cleanedDate.toLocaleDateString() +
-        " " +
-        cleanedDate.toLocaleTimeString()
+        "T" +
+        cleanedDate.toLocaleTimeString().split(" ")[0]
     )
 }
```

Comment:
The function sounds general but it's doing oddly specific like settings hours back by 1. Maybe we should have an argument as Data object?

Reply:
Wait a minute. It did have `Date` argument. Eh anyway.

Reply:
this format is accepted by django, im sending a date in string format and django accepts it and returns the correct values, i've tested it locally so now its just missing to test on qc